agents/go-agents/core/process/process.go

```diff
@@ -175,8 +176,14 @@ type processesMap struct {
 
 // Start starts MachineProcess.
 func Start(newProcess MachineProcess) (MachineProcess, error) {
-	// wrap command to be able to kill child processes see https://github.com/golang/go/issues/8854
-	cmd := exec.Command("setsid", shellInterpreter, "-c", newProcess.CommandLine)
+
+  var cmd *exec.Cmd
```

Comment:
Looks like that formatting is wrong. Please execute
```
go fmt agents/go-agents/core/process/process.go
```

Reply:
Done, thanks for suggestion